### PR TITLE
increase codec adapter max blob size to 8192 bytes

### PR DESCRIPTION
**Pull request by stolx**
Signed-off-by: Oleksandr Strelchenko <strelchenko.oleksandr@gmail.com>

### CODE REVIEW

**Comment by sofci** [comment]
Can one of the admins verify this patch?

File: src/include/sof/audio/codec_adapter/codec_adapter.h [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 #include <sof/audio/codec_adapter/codec/generic.h>
 
-#define MAX_BLOB_SIZE 4096
+#define MAX_BLOB_SIZE 8192
 /*****************************************************************************/
```
Comment (dbaluta) [inline src/include/sof/audio/codec_adapter/codec_adapter.h:14]: - Commit message must specify WHY the change is needed.
- Commit subject should start with ``audio: codec_adapter: ...``

**Comment by dbaluta** [comment]
@stolx  something went wrong, you should have only one commit.

So, when addressing feedback just squash your changes into the initial patch and then force push the patch onto your github branch.

**Comment by stolx** [comment]
hi @dbaluta 

squashed changes into one commit, thank you!

**Comment by dbaluta** [comment]
OK, looks good to me now. What is wave custom codec? Just out of curiosity.

**Review by dbaluta — APPROVED** [review]

**Review by paulstelian97 — APPROVED** [review]

**Review by lgirdwood — APPROVED** [review]

**Comment by lgirdwood** [comment]
SOFCI TEST

**Comment by stolx** [comment]
hi @dbaluta 
This is codec by Waves Audio, implements multiple algorithms on render path.

Can you suggest how to proceed with this PR now, I see some CI checks failed, not sure if they suppose to?

**Comment by lgirdwood** [comment]
@stolx I've forgotten the CI command here. @xiulipan @zrombel how do we run CI here ?
@stolx please lookout for an GH SOF invite, this means CI becomes automatic.

**Comment by xiulipan** [comment]
@lgirdwood test this please
